README.md

```diff
@@ -35,9 +35,15 @@ lang := enry.GetLanguage("foo.cpp", "<cpp-code>")
 
 Developmemt
 -----------
-*enry* re-uses parts of original [linguist](https://github.com/github/linguist) especially data in `languages.yml` to generate internal data structures. In oreder to update to latest upstream run
+*enry* re-uses parts of original [linguist](https://github.com/github/linguist)  to generate internal data structures. In order to update to latest upstream and generate the necessary code you must run:
 
-    make clean code-generate
+    go generate
+
+We update enry due to changes  in linguist's master branch related to the following files:
```

Comment:
But when are you updating this? After witch actions? Randomly, if yes document it. The people can open a request with the update? Or they can request the update?